### PR TITLE
Add sst_cs_software_management workloads

### PR DESCRIPTION
**Pull request by dmach**

### CODE REVIEW

File: configs/sst_cs_software_management-unwanted.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+document: feedback-pipeline-workload
```
Comment (asamalik) [inline configs/sst_cs_software_management-unwanted.yaml:1]: Can you please change this to:

```
document: feedback-pipeline-unwanted
```

The [exclusion list](https://github.com/minimization/content-resolver/blob/master/config_specs/exclusion.yaml) is a different config type.

Comment (dmach) [reply configs/sst_cs_software_management-unwanted.yaml]: Updated, thanks!

**Comment by asamalik** [comment]
Or you know what? I can just change it for you after the merge, that makes it easier for everyone!

**Comment by asamalik** [comment]
Oh, you did it already, never mind! Thanks!

**Pull request merged**